crawl.mjs

```diff
@@ -4,14 +4,14 @@ import { spawn } from "child_process";
 const outDir = "src/shared/translations/";
 const recommendationsFile = "recommended-instances.json";
 const instanceStatsFile = "src/shared/instance_stats.ts";
-const min_monthly_users = 5;
+const min_monthly_users = 10;
```

Comment:
Revert this change.

Reply:
Done, that was left over from testing, apologies.